js/app.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Great start of this constructor! Due pay attention to the values you are assigning tho. As of right now, none of those are instantiated, so the properties essentially have the value of null. When creating objects with specific property values, the values needed are usually passed in as a parameter.